### PR TITLE
extend linter to test files

### PR DESCRIPTION
**Pull request by erossignon**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -20,10 +21,41 @@
...
+        "import/no-dynamic-require": "off",
+        "import/no-extraneous-dependencies": "off",
+        "import/newline-after-import": "off",
+        "import/no-unresolved": "off",
+        "import/no-useless-path-segments": "off",
+        "import/order": "off",
+        "no-console": "off",
+        "no-lonely-if": "off",
+        "no-multi-assign": "off",
+        "no-undef": "off",
```
Comment (robertsLando) [inline .eslintrc:48]: Could we remove some rules here?

**Review by robertsLando — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (erossignon) [reply .eslintrc:48]: the PR will be massive so many tests file will need updating. Are you ready ?

Comment (hipstersmoothie) [reply .eslintrc:48]: We could just take this PR as is and have follow up PRs that removes flags (I could help)

Comment (robertsLando) [reply .eslintrc:48]: Ok for me, just asking :) Typescript is good but if we disable strict and other rules it becomes almost useless :laughing:

Comment (robertsLando) [reply .eslintrc:48]: We could disable one by one

**Review by robertsLando — APPROVED** [review]

**Pull request merged**